src/cli.js

```diff
@@ -289,9 +290,12 @@ program
         filter,
         keys,
         operations,
-        values
+        values,
+        cGroups
       } = options
 
+      const _cGroups = cGroups ? cGroups.split(',') : null
```

Comment:
I think it's better to change to `componentsGroups`, it is better for readability in the code.